.travis.yml

```diff
@@ -1,12 +1,18 @@
 language: objective-c
 osx_image: xcode9
+env:
+  - SCHEME="Valet iOS";SDK="iphonesimulator";DESTINATION="platform=iOS Simulator,OS=11.0,name=iPhone 7";ACTION="build test"
+  - SCHEME="Valet tvOS";SDK="appletvsimulator11.0";DESTINATION="platform=tvOS Simulator,name=Apple TV";ACTION="build test"
+  - SCHEME="Valet Mac";SDK="macosx10.13";DESTINATION="platform=OS X";ACTION="build test"
+  - SCHEME="Valet watchOS";SDK="watchos4.0";DESTINATION="";ACTION="build"
```

Comment:
How does this work? I'm assuming this is a travis feature? Is this spinning up new builds for each env?

Reply:
This is entirely a travis feature – I just learned about it yesterday. Documentation is here: https://docs.travis-ci.com/user/speeding-up-the-build/

Reply:
Basically we're setting environment variables that will be given to each parallel instance.